lib/rails/generators/tailwindcss/component_generator.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require "rails/generators/erb/component_generator"
+
+module Tailwindcss
+  module Generators
+    class ComponentGenerator < Erb::Generators::ComponentGenerator
+      include ViewComponent::AbstractGenerator
+
+      source_root File.expand_path("templates", __dir__)
+      class_option :sidecar, type: :boolean, default: false
+      class_option :inline, type: :boolean, default: false
+      class_option :stimulus, type: :boolean, default: false
+
+      def engine_name
+        "erb"
+      end
+
+      def copy_view_file
+        super
+      end
+    end
+  end
+end
```

Comment:
The tests pass when I remove this (everything I removed is inherited from `Erb::Generators::ComponentGenerator`).

```suggestion
module Tailwindcss
  module Generators
    class ComponentGenerator < Erb::Generators::ComponentGenerator
      source_root File.expand_path("templates", __dir__)
    end
  end
end
```

I'd rather keep the generator as lean as possible if its behavior is the same as the Erb generator. What do you think?

Reply:
Yup, makes perfect sense